### PR TITLE
Updating the scripts

### PR DESCRIPTION
**Pull request by tabbassidaloii**
Update 1) The schema is updated from GdbConstructImpl3 to GdbConstructImpl4 (DbBuilder.java).

Update 2) In the previous version, the user should modify BioMart2Bdb.java and add DATASOURCENAME and VERSION. In this version, the user should provide them as arguments meaning that instead of two mandatory arguments (configFile and outputPath), one should provide four arguments:

```
java -jar BioMart2BridgeDb.jar <DATASOURCENAME> <VERSION> <configFile> <outputPath> <oldDB> <inclusive>
```
I will update the README.md file if you agree with these changes.

### CODE REVIEW

File: src/main/java/org/bridgedb/creator/EnsemblREST.java [hunk 1]
```diff
@@ -51,13 +51,15 @@ public class EnsemblREST {
 
 	public static void main(String[] args) throws Exception {
 		DataSourceTxt.init(); // Initialize BrideDb data source
-
-		File dir = new File(args[0]);
-		String path = args[1];
+		
+		String DATASOURCENAME = args[0];
```
Comment (egonw) [inline src/main/java/org/bridgedb/creator/EnsemblREST.java]: Java convention is to use here `datasourceName`

File: src/main/java/org/bridgedb/creator/Main.java [hunk 2]
```diff
@@ -64,6 +64,8 @@ public static void main(String[] args) throws ClassNotFoundException, IDMapperEx
 		logInit();
 		DataSourceTxt.init(); // Initialize BrideDb data source
 		File dir = null;
+		String DATASOURCENAME = null;
```
Comment (egonw) [inline src/main/java/org/bridgedb/creator/Main.java]: See earlier comment about `datasourceName`

**Review by egonw — CHANGES_REQUESTED** [review]
Small comment (hahaha). Otherwise looks great!

**Comment by tabbassidaloii** [comment]
> Small comment (hahaha). Otherwise looks great!

Fixed :)

**Pull request merged**